Autopilot/AttitudeManager/AttitudeManager.c

```diff
@@ -911,6 +919,10 @@ void readDatalink(void){
             case SET_KIVALUES:
                 setKValues(GAIN_KI,(float*)(&cmd->data));
                 break;
+            case SET_GAINS:
+                int type = (int*) (&cmd->data);
```

Comment:
Ah, interesting. You can always fork and open a new PR. @scholvat for permission issues.